content/docs/command-reference/exp/show.md

```diff
@@ -0,0 +1,99 @@
+# exp show
+
+Display `dvc experiments` as a table, with optional formatting.
+
+> Press `q` to exit.
+
+## Synopsis
+
+```usage
+usage: dvc exp show [-h] [-q | -v] [-a] [-T] [--all-commits] [--no-pager]
+                    [--include-metrics <metrics_list>]
+                    [--exclude-metrics <metrics_list>]
+                    [--include-params <params_list>]
+                    [--exclude-params <params_list>]
+                    [--sort-by <metric/param>] [--sort-order {asc,desc}]
+                    [--no-timestamp] [--sha] [--show-json]
+                    [--precision <n>]
+```
+
+## Description
+
+Shows experiments in a detailed table which includes parameters and metrics.
+Only the experiments derived from the Git `HEAD` are shown by default but all
+experiments can be included with the `---all-commits` option. Experiments are
+sorted by timestamp by default.
+
+Your terminal will enter a paginated screen by default, which you can exit by
+typing `Q` in your keyboard. Use `--no-pager` to print the entire table at once
+instead.
+
+```dvc
+$ dvc exp show --no-pager
+┏━━━━━━━━━━━━━━━┳━━━━━━━━━━┳━━━━━━━━━┳━━━━━━━━━━━━┳━━━━━━━━━━━━━━━━━┓
+┃ Experiment    ┃ avg_prec ┃ roc_auc ┃ train.n_est┃ train.min_split ┃
+┡━━━━━━━━━━━━━━━╇━━━━━━━━━━╇━━━━━━━━━╇━━━━━━━━━━━━╇━━━━━━━━━━━━━━━━━┩
+│ workspace     │  0.56191 │ 0.93345 │ 50         │ 2               │
```

Comment:
Quick Q. Why are metrics shown first? I feel like params "happen first" in chronological order. Same Q for `exp diff` (expecting same A). Mostly curious

Reply:
To be consistent with the viewer. I'm assuming that the viewer does metrics first since the user likely wants to see the experiment "results" (i.e output metrics) first when comparing a large list of experiments.